packages/calypso-css-custom-properties/README.md

```diff
@@ -0,0 +1,11 @@
+# Calypso CSS Custom Properties
+
+This package contains a number of CSS custom properties (i.e. variables) for use in Calypso.
+Most of them are synonyms for colors defined by the [Color Studio](https://automattic.github.io/color-studio/).
+
+## Build Steps
+
+```
+npm ci
```

Comment:
We shouldn't need to tell package consumers about the build. Drop build steps or highlight how to import the relevant sass or use the css?

```suggestion
npm install
```

Reply:
I'd drop the build steps

Reply:
💥 44d6a5d93edcabc504a438b19320cbbe5fa73e8d